lib/model.js

```diff
@@ -191,19 +191,34 @@ class Model {
       self._expandAttributes(options);
```

Comment:
File mode changed, please fix it

Reply:
Whoops, seems to be a result of my development setup.I have fixed this and made changes to my config so that is doesn't happen again.